OpenBLAS blas_thread_init: pthread_create failed for thread 7 of 8: Resource temporarily unavailable
OpenBLAS blas_thread_init: RLIMIT_NPROC 4096 current, 771419 max
Traceback (most recent call last):
  File "/home/akanji/miniconda3/envs/seldnian-pre/lib/python3.8/site-packages/numpy/core/__init__.py", line 22, in <module>
    from . import multiarray
  File "/home/akanji/miniconda3/envs/seldnian-pre/lib/python3.8/site-packages/numpy/core/multiarray.py", line 12, in <module>
    from . import overrides
  File "/home/akanji/miniconda3/envs/seldnian-pre/lib/python3.8/site-packages/numpy/core/overrides.py", line 7, in <module>
    from numpy.core._multiarray_umath import (
ImportError: PyCapsule_Import could not import module "datetime"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "experiment/experiment_s_p.py", line 1, in <module>
    from seldonian.seldonian import *
  File "/home/akanji/seldonian-fairness/seldonian/seldonian.py", line 1, in <module>
    from sklearn.metrics import log_loss
  File "/home/akanji/miniconda3/envs/seldnian-pre/lib/python3.8/site-packages/sklearn/__init__.py", line 80, in <module>
    from .base import clone
  File "/home/akanji/miniconda3/envs/seldnian-pre/lib/python3.8/site-packages/sklearn/base.py", line 17, in <module>
    import numpy as np
  File "/home/akanji/miniconda3/envs/seldnian-pre/lib/python3.8/site-packages/numpy/__init__.py", line 140, in <module>
    from . import core
  File "/home/akanji/miniconda3/envs/seldnian-pre/lib/python3.8/site-packages/numpy/core/__init__.py", line 48, in <module>
    raise ImportError(msg)
ImportError: 

IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!

Importing the numpy C-extensions failed. This error can happen for
many reasons, often due to issues with your setup or how NumPy was
installed.

We have compiled some common reasons and troubleshooting tips at:

    https://numpy.org/devdocs/user/troubleshooting-importerror.html

Please note and check the following:

  * The Python version is: Python3.8 from "/home/akanji/miniconda3/envs/seldnian-pre/bin/python"
  * The NumPy version is: "1.19.2"

and make sure that they are the versions you expect.
Please carefully study the documentation linked above for further help.

Original error was: PyCapsule_Import could not import module "datetime"

